content/applications/websites/livechat/chatbots.rst

```diff
@@ -8,8 +8,8 @@ response, and lead them through a series of questions and answers the same way a
 would.
 
 Chatbots can be customized to fill various roles, from customer support, to creating leads, to
-collecting contact information. The goal of the chatbot depends on the page of the website they are
-assigned, and the messages included in the script, among other criteria.
+collecting contact information. The goal of each chatbot can vary based on several criteria,
```

Comment:
This might be beyond silly, but I just realized when testing a chatbot, the chatbot works in which apps? In testing, it sends me to a website- but it can also work just in the database- and I assume the portal? I honestly don't know- and just thought of it- so that could be mentioned somewhere here the various places it can- and cannot- appear? Just an idea.

Reply:
Chatbots can be added to any livechat channel, so they can appear on any web page where a channel is added.